source/_layouts/header.blade.php

```diff
@@ -9,7 +9,7 @@
                 <li><a href="{{ locale_path($page, $page->baseUrl) }}#why-us">Soluções</a></li>
                 <li><a href="{{ locale_path($page, $page->baseUrl) }}#clients">Clientes</a></li>
                 <li><a href="{{ locale_path($page, $page->baseUrl) }}#apoie">Apoie</a></li>
-                <li><a href="{{ locale_path($page, $page->baseUrl) }}#contact">Contato</a></li>
+                <li><a href="{{ locale_path($page, $page->baseUrl) }}contact">Contato</a></li>
```

Comment:
Acho que podemos manter com a âncora e o formulário na index. O que vejo de vantagem é que poderemos rastrear a url /contato no Matomo para ver quantos acessos vieram dos qrcode que estão nos nossos impressos.

```suggestion
                <li><a href="{{ locale_path($page, $page->baseUrl) }}#contact">Contato</a></li>
```